cmd/clair-action/convert.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/quay/clair-action/output"
+	"github.com/quay/claircore"
+	"github.com/urfave/cli/v2"
+)
+
+var convertCmd = &cli.Command{
```

Comment:
any interest in writing this out to a file?

Reply:
Not really, as long as it's being written to `stdout` the caller can redirect/pipe it anywhere they want and the command (in code) is easier to understand. Let me know if I'm getting the wrong end of the stick here.